jwst/regtest/test_NIRcam_image2a.py

```diff
@@ -0,0 +1,66 @@
+""" Test for the image2 pipeline using NIRCam image data. """
```

Comment:
The intention of ticket JP-1168 was to write a test of the calwebb_detector1 (level-2a) pipeline, not the calwebb_image2 (level-2b) pipeline. So all references to image2 and ``Image2Pipeline`` need to be changed to ``Detector1Pipeline``. That also means the input needs to be an uncal product (instead of rate) and the outputs to be tested need to be _rate and _rateints. Alternatively, this code could be turned into satisfying JP-1179, which as intended to be a test of the ``Image2Pipeline`` on NIRCam images.